packages/react-components/react-utilities/.swcrc

```diff
@@ -10,9 +10,6 @@
     "/**/*.test.tsx"
   ],
   "jsc": {
-    "experimental": {
-      "plugins": [["swc-plugin-de-indent-template-literal", {}]]
```

Comment:
Looks like we'll need to fix the de-indent plugin?

Reply:
https://github.com/bsunderhus/swc-plugin-de-indent-template-literal/issues/1

Reply:
Just published v1.4.0 of the plugin (https://www.npmjs.com/package/swc-plugin-de-indent-template-literal/v/1.4.0), it should be compatible with `swc/core@1.4.0`

Reply:
works ! thx for quick fix !